### PR TITLE
spacing in logmessage

### PR DESCRIPTION
**Pull request by erwindon**
a specific log message lists a few terms. 2 of these terms do not have any spacing between them as I noticed in actual output,
2 additional spacing adjustments for consistency.

### CODE REVIEW

File: artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/services/toolbar.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ var Artemis;
                     <form class="toolbar-pf-actions">
                         <div class="form-group toolbar-pf-filter">
                             <div class="input-group">
-                                <div class="input-group-btn">
+                                <div class="input-group-btn" style="padding-left: 10px">
```
Comment (clebertsuconic) [inline artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/services/toolbar.js:28]: @andytaylor can you review this?

**Comment by brusdev** [comment]
If you are applying the same style `padding-left: 10px` to all divs with the `input-group-btn` class you could define that class in the https://github.com/apache/activemq-artemis/blob/master/artemis-hawtio/activemq-branding/src/main/webapp/plugin/css/activemq.css

File: artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/TopologyMemberImpl.java [hunk 2]
```diff
@@ -154,7 +154,7 @@ public URI toBackupURI() {
 
    @Override
    public String toString() {
-      return "TopologyMember[id = " + nodeId + ", connector=" + connector + ", backupGroupName=" + backupGroupName + ", scaleDownGroupName=" + scaleDownGroupName + "]";
+      return "TopologyMember[id=" + nodeId + ", connector=" + connector + ", backupGroupName=" + backupGroupName + ", scaleDownGroupName=" + scaleDownGroupName + "]";
```
Comment (brusdev) [inline artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/TopologyMemberImpl.java:157]: Is this change on purpose?

**Comment by erwindon** [comment]
I have to apologize deeply here!
Due to a faulty rebase, this PR contained also code from another PR.
None of the HTML changes were supposed to be here.
That has now been corrected.

File: artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/services/toolbar.js (hunk 1 above)
Comment (erwindon) [reply artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/services/toolbar.js]: sorry, the HTML code should not have been here, see comment below

File: artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/TopologyMemberImpl.java (hunk 2 above)
Comment (erwindon) [reply artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/impl/TopologyMemberImpl.java:157]: yes, though cosmetic. the other items in the same message do not have spacing around the "="
